internal/command/postgres/create.go

```diff
@@ -590,3 +594,20 @@ func resolveForkFromManager(ctx context.Context, machines []*api.Machine) string
 
 	return flypg.StolonManager
 }
+
+func resolveImageFromForkVolume(vol *api.Volume, machines []*api.Machine) string {
+	// If the volume is not associated with a machine, we can't resolve the image and wil
+	// fallback to the default image.
+	if len(machines) == 0 {
+		return ""
+	}
```

Comment:
not needed ?  the iteration over an empty `machines` array will fallback to last `return ""`